### ISSUE TITLE
Links to figures not working

### ISSUE THREAD

**Issue by HannoSpreeuw** [carpentries-incubator/lesson-gpu-programming#72]
Links to figures do not yet point to the `fig` directory and are therefore not working.

**Comment by isazi** [comment]
The new figures? The old ones are working for me.

**Comment by HannoSpreeuw** [comment]
E.g. the link in episode `02-cupy.md` to `Example of convolution` does not work.
So let me rephrase the issue, the link does not seem to be able to find the `fig` directory when I launch `02-cupy.md` using `grip -b 02-cupy.md` and then click on `Example of convolution`.

This is the code in `02-cupy.md`:
```
![Example of convolution](../fig/2D_Convolution_Animation.gif)
```
and that seems the correct path.

It does render correctly in this repo.

**Comment by HannoSpreeuw** [comment]
Can anyone reproduce this? I am still puzzled.

**Comment by HannoSpreeuw** [comment]
We think this is a limitation of grip, so we are not going to solve it.
It works if the figures are in the same directory as the .md files, but we
don't want that, because it will become messy.
